packages/taucmdr/cli/commands/experiment/list.py

```diff
@@ -69,6 +69,13 @@ def main(self, argv):
         """
         args = self._parse_args(argv)
 
+        try:
+            proj_id = Project.controller().selected().eid
+            _filter = lambda x: [e for e in x if e['project'] == proj_id]
+            print("Listing experiments of selected project")
```

Comment:
Now that I think of it, I left that piece of code to make it obvious only a subset of the experiments are displayed. You might want to change or format it to fit better in the interface.